style.css

```diff
@@ -106,8 +110,118 @@
     .btn{
         text-align: left;
         margin-top: 38px;
-        /*margin-right: 0;*/
-        /*margin-left: 20px;*/
+        margin-left: 20px;
         border: none;
     }
+}
+.recent_posts{
+    margin-top: 59px;
+    background-color: #EDF7FA;
+    margin-bottom: 38px;
+}
+.recent_heading{
+    display: flex;
+}
+.view{
+    display: none;
+}
+.post{
+    text-align: center;
+    font-family: 'Heebo', sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    color: #21243D;
+    margin-left: 135px;
+    margin-right: 135px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+.post_one{
+    background-color: white;
+    margin-left: 11px;
+    margin-right: 10px;
+    margin-bottom: 38px;
+}
+.one_heading{
+    color:#21243D;
+    font-family: 'Heebo';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 30px;
+    text-align: left;
+    width: 327px;
+    height: 60px;
+    margin-left: 19px;
+    padding-top: 12px;
+}
+.date_info{
+    display: flex;
+    flex-direction: row;
+    margin-top: 12px;
+}
+.date{
+    font-family: 'Heebo', sans-serif;
```

Comment:
This looks like a primary font. So instead of applying it on all the elements one by one just put this on body tag. When you want to apply some other font then only use this property.

Reply:
Yes, I thought this after repeatedly using it again and again. Will take care from next time.